data/campaigns/Legend_of_Wesmere/utils/low-macros.cfg

```diff
@@ -59,13 +59,16 @@
 #enddef
 
 #define ELVES_SCOUTS
-Elvish Scout, Elvish Rider, Elvish Outrider, Elvish Horse Archer#enddef
+Elvish Scout, Elvish Rider, Elvish Outrider
+#enddef
 
 #define ELVES_STEEL
-Elvish Fighter, Elvish Hero, Elvish Captain, Elvish Marshal, Elvish Champion, Elvish Archer, Elvish Marksman, Elvish Sharpshooter, Elvish Ranger, Elvish Avenger, {ELVES_SCOUTS}#enddef
+Elvish Fighter, Elvish Hero, Elvish Captain, Elvish Marshal, Elvish Champion, Elvish Archer, Elvish Marksman, Elvish Sharpshooter, Elvish Ranger, Elvish Avenger, {ELVES_SCOUTS}
+#enddef
 
 #define ELVES_NON_STEEL
-Elvish Shaman, Elvish Shyde, Elvish Enchantress, Elvish Sorceress, Elvish Sylph, Elvish Druid, Wose, Elder Wose, Ancient Wose, {ELVES_SCOUTS}#enddef
+Elvish Shaman, Elvish Shyde, Elvish Enchantress, Elvish Sorceress, Elvish Sylph, Elvish Druid, Wose, Elder Wose, Ancient Wose, {ELVES_SCOUTS}
+#enddef
```

Comment:
You _could_ be right about the scenario 5 one, but when I looked at it it seemed like a case of incorrect usage. Since it was already that way, if you don't feel like fixing it, it's probably not all that important. (It would only be a problem if it's used in the middle of a line, for example like the various SPECIAL_NOTE* macros.)